cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -907,6 +909,7 @@ SwiftGenerator::writeMemberDoc(IceInternal::Output& out, const DataMemberPtr& p)
 
     if (!doc.misc.empty())
     {
+        out << nl << "///";
```

Comment:
Added an extra newline to the comment, to be consistent with how we generate other sections. Also, I just think it looks better.